### PR TITLE
Document how to use the repo with other build systems

### PR DESCRIPTION
**Pull request by stagnation**
Some notes on how to use this deployment with other build systems. Following a recent discussion about pants in slack.

### CODE REVIEW

**Comment by stagnation** [comment]
I did not verify that the pants section works, as I do not currently have a pants project setup.

File: README.md [hunk 1]
```diff
@@ -92,6 +92,47 @@ bazel build --config=remote-ubuntu-22-04 @abseil-hello//:hello_main
...
+[GLOBAL]
+remote_cache_read = true
+remote_cache_write = true
+remote_store_address = "grpc://localhost:8980"
+remote_execution_address = "grpc://localhost:8980"
+remote_execution = true
+remote_instance_name = "fuse"
+remote_execution_extra_platform_properties = [
+  "OSFamily=linux",
+  "container-image=docker://ghcr.io/catthehacker/ubuntu:act-22.04@sha256:5f9c35c25db1d51a8ddaae5c0ba8d3c163c5e9a4a6cc97acd409ac7eae239448",
```
Comment (Geethree) [inline README.md:117]: This is dependent on the `container-image` property which is keyed to the worker's image digest sha. When I was testing I had to specify both the `OSFamily` and `container-image` properties to get the scheduler to pick up the run.

Comment (moroten) [reply README.md:117]: @Geethree It would be great to add an example configuration and a test. Would you like to contribute with this?

**Comment by stagnation** [comment]
Are we happy with this as-is?

**Comment by moroten** [comment]
I'll have a look when I'm back in Tuesday if it's ready to merge.

**Pull request merged**